packages/docusaurus-theme-common/src/utils/structuredDataUtils.ts

```diff
@@ -45,6 +48,7 @@ function getBlogPost(
     name: title,
     description,
     datePublished: date,
+    dateModified: convertDate(lastUpdatedAt),
```

Comment:
```js
const dateModified = lastUpdatedAt ? convertDate(lastUpdatedAt) : undefined;
```

```suggestion
    ...(dateModified ? {dateModified} : {}),
```

Need to only add it conditionally?